packages/lisk-transactions/src/errors.ts

```diff
@@ -87,9 +87,9 @@ export const convertToTransactionError = (
 	return errors.map(
 		error =>
 			new TransactionError(
-				`'${error.dataPath}' ${error.message}`,
+				`'${error.dataPath || '.asset'}' ${error.message}`,
```

Comment:
At this point, I think non existent `dataPath` doesn't necessarily mean `.asset`